packages/netsuite-adapter/src/client/deploy_xml_utils.ts

```diff
@@ -69,11 +60,9 @@ export const reorderDeployXml = (
     .filter(node => !nodesInCycle.has(node.id))
     .map(node => node.value.customizationInfo)
 
-  const [fileCabinetCustInfos, customTypeInfos] = _.partition(
-    custInfosInTopologicalOrder, isFileCabinetCustomType
-  )
+  const customTypeInfos = custInfosInTopologicalOrder.filter(custInfo => !isFileCabinetCustomType(custInfo))
```

Comment:
nit- with removing `filter(isCustomTypeInfo)` from line 71 & 74
```suggestion
  const customTypeInfos = custInfosInTopologicalOrder.filter(isCustomTypeInfo)
```